subprojects/model-groovy/src/integTest/groovy/org/gradle/cli/TaskOptionsSpec.groovy

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.cli
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+
+class TaskOptionsSpec extends AbstractIntegrationSpec {
+    def "can use a built-in option"() {
+        when:
+        buildScript """
+            abstract class MyTask extends DefaultTask {
+                @Option(option="profile", description="foobar")
+                void setProfile(String value) {
+                    System.out.println("profile=" + value)
+                }
+            }
+
+            tasks.register('mytask', MyTask.class)
+        """
+
+        then:
+        succeeds "mytask", "--profile"
```

Comment:
Should assert that the built in profile option was used (it should produce a line of output), or that the task option did **not** run with `!output.contains "profile=myvalue"`

Reply:
I think "See the profiling report at" should do this nicely. Does that seem OK to you?

Reply:
That should be enough, yes.